assets/css/custom.css

```diff
@@ -0,0 +1,24 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+}
+
+.item {
+  text-align: center;
+  margin-bottom: 0.5em;
+}
+
+.item label {
+  text-transform: uppercase;
+  font-weight: bold;
+}
+
+.foto {
```

Comment:
- Tamanho: é bom sempre que você for por tamanho em alguma coisa usar as [unidades](https://www.w3schools.com/cssref/css_units.php) relativas(em, rem, vw....). Nesse caso eu recomendo 10vw. Onde o vw vai ser relativo ao tamanho da tela.
- Borda: é bom por ela direto na imagem, que a imagem tbm ganha os cantos arredondados, e encaixa melhor com a borda